scalabel/label/io.py

```diff
@@ -47,6 +48,8 @@ def parse(raw_frame: DictStrAny, validate_frames: bool = True) -> Frame:
                     label.poly2d = [
                         Poly2D.construct(**p) for p in label.poly2d  # type: ignore # pylint: disable=line-too-long
                     ]
+                if label.graph is not None:
+                    label.graph = Graph.construct(**label.graph.dict())
```

Comment:
Do not add .dict() here. I know mypy complains, but this is because it can't recognize the special case of pydantic frame loading with validate set to false. In this setting, label.graph will stay a dict and therefore the function will crash if you call .dict() on it. Please make a test run verifying your changes work as intended before we merge.

Reply:
Got it, thanks for catching the bug. My previous test case didn't include the case where `validate=False`, so I wasn't aware of the issue. I updated the test case as well.